tests/ops/qubit/test_matrix_ops.py

```diff
@@ -1223,15 +1236,16 @@ def circuit(input_matrix):
                 range(2),
             ),
             ([[0.1, 0.2, 0.3], [0.3, 0.4, 0.2], [0.1, 0.2, 0.3]], range(3)),
+            (csr_matrix([[0.1, 0.2, 0.3], [0.3, 0.4, 0.2], [0.1, 0.2, 0.3]]), range(3)),
```

Comment:
I'd suggest modifying this (and related) tests to avoid keeping the sparse matrix fully dense --- we want to make sure the lack of entries in the sparse rep doesn't cause problems. I'd recommend looking into the (data, index, indptr) constructor for the sparse rep and using this to build some additional examples.

Reply:
True. I'll make a better test for sparse case here later.